### ISSUE TITLE
Add aria-disabled="true" to disabled buttons in list navigation

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#158]
### Problem

[molecules/molecules.navigation](https://a-ui.github.io/core_branding_scss/#molecules-navigation)

The button 'Albert Eistein' is only visually styled as disabled using CSS cursor: not-allowed. A screenreader user can't know it's disabled.

```html
<li><button class="is-disabled">Albert Einstein</button></li>
```

```css
.m-nav-list > li > a.is-disabled, .m-nav-tabs > li > a.is-disabled, .m-nav-list > li > a.is-disabled:hover, .m-nav-tabs > li > a.is-disabled:hover, .m-nav-list > li > a.is-disabled:active, .m-nav-tabs > li > a.is-disabled:active, .m-nav-list > li > button.is-disabled, .m-nav-tabs > li > button.is-disabled, .m-nav-list > li > button.is-disabled:hover, .m-nav-tabs > li > button.is-disabled:hover, .m-nav-list > li > button.is-disabled:active, .m-nav-tabs > li > button.is-disabled:active {
    background-color: #f3f3f3;
    color: #6f6f6f;
    cursor: not-allowed;
    text-decoration: none;
```

![screenshot nav list](https://user-images.githubusercontent.com/870049/98676661-2c2c5b80-235c-11eb-8e4b-8d733fc72186.jpg)

### Solution

Add `aria-disabled="true"` to communicate it's state to screenreaders.

### Articles en links

[WAI-ARIA 1.2: aria-disabled (state)](https://www.w3.org/TR/wai-aria-1.2/#aria-disabled)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
